mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule VerkWeb.Mixfile do
 
   def application do
     [mod: {VerkWeb, []},
-     env: [{VerkWeb.Endpoint, [http: [port: 4000], cache_static_manifest: "priv/static/manifest.json", server: true]}],
+     env: [{VerkWeb.Endpoint, [http: [port: 4000], server: false]}],
```

Comment:
I see... While it digesting assets is the right thing for most public facing applications, it is probably not strictly required for dashboards in admin areas... And if we want to allow github dependency definition like ({:verk_web, github: "edgurgel/verk_web", ref: "cutting-edge"} we'd have to check-in a lot of static files into git (compiled assets, minified assets, minified + gzipped assets...) everytime we change a thing... That's a lot of churn. 

I'd like to keep things as simple + straight-forward as possible. Just checking compiled files as plaintext files might not be the most advanced way, but it keeps the dev + installation process  very simple. If we encounter any real life  problems with that approach, we can always still add digesting + static manifest back. Nothing beats KISS for maintainability.